<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApolloLink</class>
 <widget class="QMainWindow" name="ApolloLink">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>546</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Directorio Telef√≥nico</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/resources/app.png</normaloff>:/resources/app.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Contactos</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Buscar:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inBuscar">
            <property name="placeholderText">
             <string>Ingresa un texto para buscar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbtnNombre">
            <property name="text">
             <string>&amp;Nombre</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbtnCargo">
            <property name="text">
             <string>&amp;Cargo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbtnUnidad">
            <property name="text">
             <string>&amp;Unidad</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnBuscar">
            <property name="text">
             <string>Buscar todos</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tblContactos">
         <property name="styleSheet">
          <string notr="true">selection-background-color: rgb(255, 250, 175);
selection-color: rgb(4, 4, 4);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionEliminar"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNuevo"/>
   <addaction name="actionGuardar"/>
   <addaction name="actionEliminar"/>
  </widget>
  <action name="actionNuevo">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/resources/agregar-usuario.png</normaloff>:/resources/agregar-usuario.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset theme="appointment-new" resource="recursos.qrc">
     <normaloff>:/resources/salvar.png</normaloff>:/resources/salvar.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionEliminar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/resources/papelera-de-reciclaje.png</normaloff>:/resources/papelera-de-reciclaje.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Eliminar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>inBuscar</tabstop>
  <tabstop>rbtnNombre</tabstop>
  <tabstop>rbtnCargo</tabstop>
  <tabstop>rbtnUnidad</tabstop>
  <tabstop>btnBuscar</tabstop>
  <tabstop>tblContactos</tabstop>
 </tabstops>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSalir</sender>
   <signal>triggered()</signal>
   <receiver>ApolloLink</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
